site/en/blog/watch-video-using-picture-in-picture/index.md

```diff
@@ -4,7 +4,7 @@ title: Watch video using Picture-in-Picture
 description: >
   Control Picture-in-Picture for video elements on your website.
 date: 2018-10-19
-updated: 2022-07-20
+updated: 2022-08-08
```

Comment:
```suggestion
```

Reply:
https://web.dev/handbook/yaml-front-matter/#updated

Reply:
Thanks!

Reply:
Can you please commit this.

Reply:
I already updated file. See https://github.com/GoogleChrome/developer.chrome.com/pull/3403/files

Reply:
Stupid GitHub. 😖